git_machete/client.py

```diff
@@ -2048,9 +2048,16 @@ def create_github_pr(
         reviewers: List[str] = utils.get_non_empty_lines(utils.slurp_file_or_empty(reviewers_path))
         if reviewers:
             print(
-                fmt(f'Adding {", ".join(f"`{reviewer}`" for reviewer in reviewers)} as reviewer{"s" if len(reviewers) > 1 else ""} to PR #{pr.number}...'),
+                fmt(f'Adding {", ".join(f"`{reviewer}`" for reviewer in reviewers)} as reviewer{"s" if len(reviewers) > 1 else ""} to PR #{pr.number}...\n'),
                 end='', flush=True)
-            add_reviewers_to_pull_request(org, repo, pr.number, reviewers)
+            try:
+                add_reviewers_to_pull_request(org, repo, pr.number, reviewers)
+            except UnprocessableEntityHTTPError as e:
+                if 'Reviews may only be requested from collaborators.' in e.msg:
+                    warn(f"There are some invalid reviewers in .{self.__git.get_git_subpath('info', 'reviewers')} file.\n"
```

Comment:
Stray period, probably
```suggestion
                    warn(f"There are some invalid reviewers in {self.__git.get_git_subpath('info', 'reviewers')} file.\n"
```